src/openzaak/components/catalogi/models/resultaattype.py

```diff
@@ -145,7 +145,9 @@ class ResultaatType(ETagMixin, OptionalGeldigheidMixin, models.Model):
         blank=True,
         help_text=_(
             "Naam van de attribuutsoort van het procesobject dat "
-            "bepalend is voor het einde van de procestermijn."
+            "bepalend is voor het einde van de procestermijn. "
+            "In het geval van afleidingswijze zaakobject kunnen nested attributen "
+            "gespecificeerd worden door middel van `/` (bijv. `some/nested/path`)"
```

Comment:
But you expect `.` not `/` as a separator?

Reply:
I split by `/` to create a path, so nested attributes are specified with `/`: https://github.com/open-zaak/open-zaak/pull/1683/files#diff-4edd9bb7115bb5ddd0eb9986700a0fad6adf34a7d519b394b85359c220d292a5R146

I also added tests for `.` to check if we do still support attributes with dots in them:

```
{
    "some.attribute": "foo"
}
```